ansible/roles/cyhy_mailer/tasks/main.yml

```diff
@@ -41,13 +41,13 @@
       region = {{ ses_aws_region }}
       role_arn = {{ ses_send_email_role }}
 
-# docker-compose will automatically use docker-compose.yml and
+# docker compose will automatically use docker-compose.yml and
```

Comment:
This is phrased awkwardly to me now so I thought this might be a bit clearer.

```suggestion
# The compose command will automatically use docker-compose.yml and
```